### PR TITLE
Dao page 1

### PR DESCRIPTION
**Pull request by fairlighteth**
- Adds a DAO page on /daos

Screenshot 2023-07-28 at 13 32 19

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @fairlighteth , great!
However, the issue with scrolling the page after menu opening in iOS is still reproducible. 
Also, the page looks this way when open it in iPhone: 
image
image

When I open the page, the animation starts running to fast for me making me think that the page has not been fully loaded (very personal opinion) :)
image

**Comment by fairlighteth** [comment]
@elena-zh Did some updates to resolve the layout issues. Could you double check?

File: modules/utm/utils.ts [hunk 1]
```diff
@@ -50,32 +50,40 @@ export function hasUtmCodes(utm: UtmParams | undefined): boolean {
 }
 
 export function addUtmToUrl(href: string, utm: UtmParams): string {
-  const origin = typeof window !== 'undefined' ? window.location.origin : CONFIG.url.root
-  const url = new URL(href, origin)
+  const origin = typeof window !== 'undefined' ? window.location.origin : CONFIG.url.root;
+  const url = new URL(href, origin);
 
+  // Extract the hash and its associated query parameters
+  const [hashPath, hashQuery] = url.hash.split('?');
```
Comment (alfetopito) [inline modules/utm/utils.ts:57]: Can you check your linter? It's adding semi-colons

**Review by shoom3301 — APPROVED** [review]
Briefly checked the code and mostly tested the appearance.
Looks dope!

File: .babelrc [hunk 2]
```diff
@@ -1,4 +1,11 @@
 {
   "presets": ["next/babel"],
+  "env": {
+    "production": {
+      "plugins": [
+        ["styled-components", { "ssr": true, "displayName": false}]
+      ]
+    }
+  },
   "plugins": [["styled-components", { "ssr": true, "displayName": true }]]
```
[the remark this answers is not in the document]
Comment (fairlighteth) [reply .babelrc:10]: Saw it suggested somewhere as a fix for...don't recall anymore, heh.

File: components/Home/index.tsx [hunk 3]
```diff
@@ -30,13 +28,19 @@ export interface HomeProps {
...
+    const elem = document.getElementById(targetId);
+    elem?.scrollIntoView({ behavior: "smooth" });
+  }
 
   return (
     <>
       {/* Hero */}
       <Section className="container" hero>
-        <SectionContent>
+        <SectionContent margin={'12rem auto'}>
```
[the remark this answers is not in the document]
Comment (fairlighteth) [reply components/Home/index.tsx:43]: No sir, this SectionContent component is reused in many places and needs optional flexibility like this.

**Pull request merged**